ee/tabby-webserver/src/repositories/mod.rs

```diff
@@ -12,23 +12,28 @@ use axum::{
 };
 use hyper::Body;
 use juniper_axum::extract::AuthBearer;
-use tabby_common::path::repositories_dir;
+use tabby_common::config::Config;
 use tracing::{instrument, warn};
 
 use crate::{
-    repositories,
-    repositories::resolve::{resolve_dir, resolve_file, resolve_meta, Meta, ResolveParams},
+    repositories::resolve::{
+        resolve_all, resolve_dir, resolve_file, resolve_meta, Meta, ResolveParams,
+    },
     schema::ServiceLocator,
 };
 
-pub fn routes(locator: Arc<dyn ServiceLocator>) -> Router {
+pub fn routes(conf: Arc<Config>, locator: Arc<dyn ServiceLocator>) -> Router {
     Router::new()
+        .route("/resolve/", routing::get(resolve))
```

Comment:
will `/resolve` resolve to this path?

Reply:
no, it causes `404`. I added both paths (with & without) to prevent this issue.